package.json

```diff
@@ -20,10 +20,12 @@
     "clsx": "^1.2.1",
     "prism-react-renderer": "^1.3.5",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "react-scripts": "^5.0.1"
   },
   "devDependencies": {
-    "@docusaurus/module-type-aliases": "2.3.1"
+    "@docusaurus/module-type-aliases": "2.3.1",
+    "tailwindcss": "^3.3.2"
```

Comment:
Actually, I added tailwindcss as a dependency. So, other devs should be able to use it after running `npm install`